modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLViewFileEntryTypesDisplayContext.java

```diff
@@ -73,7 +73,8 @@ public CreationMenu getCreationMenu() {
 						"/document_library/edit_file_entry_type", "redirect",
 						PortalUtil.getCurrentURL(_httpServletRequest));
 					dropdownItem.setLabel(
-						LanguageUtil.get(_httpServletRequest, "new"));
+						LanguageUtil.get(
```

Comment:
Q!

This is ok but
Why didn't you use the same way?

```
LanguageUtil.format(
		_httpServletRequest, "new-x",
		LanguageUtil.get(
			_httpServletRequest, "document-type"),false)
```

There is a reason?

Reply:
This is because the variable `new-document-type` already exists, but there is no `new-metadata-set`, so here we can just use it, and in the other change we can create it as shown.

Reply:
My question is to make it more ... uniform so we can use the 
```
LanguageUtil.format(
		_httpServletRequest, "new-x",
		LanguageUtil.get(
			_httpServletRequest, "document-type"),false)
```
because all the variables exists and is like the other menu creation

Reply:
I understand, I'll change it :)